gripper_effort_controller:
  type: effort_controllers/GripperActionController
  gains:
    finger_joint: {p: 1, d: 0, i: 0.1, i_clamp: 1}
  joint: finger_joint #important to call it joinT in the singular! action_monitor_rate: 20 goal_tolerance: 0.002 max_effort: 100 stall_velocity_threshold: 0.001 stall_timeout: 1.0

gripper_position_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - finger_joint
  constraints:
    goal_time: 0.6
    stopped_velocity_tolerance: 0.05
    gripper_finger_joint: {trajectory: 0.2, goal: 0.2}
  stop_trajectory_duration: 0.5
  state_publish_rate:  125
  action_monitor_rate: 10

gazebo_ros_control:
  pid_gains:
    joint2: {p: 100.0, i: 0.01, d: 10.0}
